deps/v8/gypfiles/features.gypi

```diff
@@ -113,6 +113,13 @@
 
     'v8_enable_fast_mksnapshot%': 0,
   },
+  'conditions': [
+    ['(OS=="aix") or (OS=="win" and (clang!=1 or v8_target_arch=="ia32"))', {
```

Comment:
Should embedded builtins only be disabled for 32-bit Intel on Windows only or on all 32-bit Intel platforms? (I know we only support 32-bit Intel on Windows for Node.js but thinking of others).

Reply:
I'll replicate the predicate in BUILD.gn:
https://github.com/nodejs/node/blob/ae461f52330710858f1282bfacdd2da6e0145dd9/deps/v8/BUILD.gn#L74-L77

Reply:
For reference, the condition in [BUILD.gn](https://github.com/v8/v8/blob/fb4d8c01afe0ffeb1959d389fae65c4c35e62709/BUILD.gn#L74-L77):

```gn
  # Enable embedded builtins.
  # TODO(jgruber,v8:6666): Support ia32 and maybe MSVC.
  v8_enable_embedded_builtins = v8_use_snapshot && v8_current_cpu != "x86" &&
                                !is_aix && (!is_win || is_clang)
```